federation/pkg/federation-controller/util/eventsink/eventsink.go

```diff
@@ -37,14 +44,80 @@ func NewFederatedEventSink(clientset fedclientset.Interface) *FederatedEventSink
 	}
 }
 
-func (fes *FederatedEventSink) Create(event *api_v1.Event) (*api_v1.Event, error) {
-	return fes.clientset.Core().Events(event.Namespace).Create(event)
+// TODO this is uses a reflection conversion path and is very expensive.  federation should update to use client-go
```

Comment:
Can you make an issue out of this and CC somebody from the federation team here?